packages/wpcom.js/src/lib/me.js

```diff
@@ -70,6 +70,17 @@ Me.prototype.sites = function ( query, fn ) {
 	return this.wpcom.req.get( '/me/sites', query, fn );
 };
 
+/**
+ * A list of the current user's sites plugins
+ *
+ * @param {object} [query] - query object parameter
+ * @param {Function} fn - callback function
+ * @returns {Function} request handler
+ */
+Me.prototype.sitesPlugins = function ( query, fn ) {
```

Comment:
We no longer introduce new methods to `wpcom.js`. Could you just directly call `wpcom.req.get()` instead?

See p4TIVU-9KX-p2 and p4TIVU-9Tk-p2 for additional context.